index.bs

```diff
@@ -3190,6 +3190,22 @@ value).</p>
 
    </dd>
 
+   <dt><dfn>HTML character reference in annotation state</dfn></dt>
+
+   <dd>
+
+    <p>Attempt to <a>consume an HTML character reference</a>, with the <a spec=html>additional
```

Comment:
Yeah the only difference from "no additional allowed character" is whether `&>` produces a parse error or not. WebVTT doesn't take note of the parse error, so for WebVTT there is technically no difference at all, currently. WebVTT's syntax doesn't allow lone ampersands, so it is a syntax violation regardless.